src/main/kotlin/oop/Observer2/Roulette.kt

```diff
@@ -0,0 +1,12 @@
+package oop.Observer2
+
+import kotlin.properties.Delegates
+
+class Roulette{
+
+  var observer: CategoryObserver? = null
```

Comment:
There is no need to create a separated object in this case. I think it is more natural if Players are observing the Roulette directly 👍